doc/source/examples/similarity_explanations_mnist.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<p>Selecting a reference -&gt; Select a refenrece</p><p>is performed in oreder to speed -&gt; is performed to speed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>